### PR TITLE
PEP8 formatted and linter.sh file

### PR DESCRIPTION
**Pull request by ChristopherBilg**

### CODE REVIEW

File: opt_probs.py [hunk 1]
```diff
@@ -275,7 +276,7 @@ def eval_node_probs(self):
         # Get probs
         probs = np.zeros([self.length, self.max_val, self.max_val])
 
-        if len(self.keep_sample) == 0:
+        if not self.keep_sample:
```
Comment (gkhayes) [inline opt_probs.py]: This line causes the following error on my machine:
`ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`
Could we revert back to the original version?
```suggestion
        if len(self.keep_sample) == 0:      
```

File: opt_probs.py [hunk 2]
```diff
@@ -289,7 +290,7 @@ def eval_node_probs(self):
                 subset = self.keep_sample[np.where(
                     self.keep_sample[:, parent[i - 1]] == j)[0]]
 
-                if len(subset) == 0:
+                if not subset:
```
Comment (gkhayes) [inline opt_probs.py]: As above, this line is also causing an error when run on my machine. Could we also revert back here?
```suggestion
                if len(subset) == 0:
```

File: test.py [hunk 3]
```diff
@@ -3,16 +3,9 @@
...
 import unittest
-from fitness import OneMax, FlipFlop, head, tail, max_run, Queens, np
-
-# from algorithms import *
-# from discrete import *
-# from fitness import *
-# from decay import *
-# from neural import *
-# import numpy as np
+from fitness import (OneMax, FlipFlop, head, tail, max_run, Queens, np,
```
Comment (gkhayes) [inline test.py]: `np` is numpy, not a function\class created in `fitness`. As a result, we should import that with a separate import statement.
```suggestion
import numpy as np
from fitness import (OneMax, FlipFlop, head, tail, max_run, Queens, 
```

Comment (ChristopherBilg) [reply test.py]: Good catch, I see that now.

File: test.py [hunk 4]
```diff
@@ -22,50 +15,51 @@ class TestFitness(unittest.TestCase):
...
     @staticmethod
     def test_queens():
         """Test Queens fitness function"""
         state = np.array([1, 4, 1, 3, 5, 5, 2, 7])
-        assert(Queens().evaluate(state) == 6)
+        assert Queens().evaluate(state) == 6
 
-
-def test_max_k_color(self):
+    @staticmethod
```
Comment (gkhayes) [inline test.py]: Is it necessary to have the `@staticmethod` decorators? They aren't used in the example included in the `unittest` documentation: [https://docs.python.org/3/library/unittest.html](url)

Comment (gkhayes) [reply test.py]: OK. Thanks for explaining that. I haven't used @staticmethod before in Python, and don't have much experience with Java, so was uncomfortable with their use.

**Pull request merged**